packages/app-project/stores/User/UserPersonalization/UserPersonalization.js

```diff
@@ -46,10 +46,8 @@ const UserPersonalization = types
   }))
   .actions(self => {
     function _onUserChange() {
-      const { project, user } = getRoot(self)
-      if (user.id) {
-        self.notifications.fetchAndSubscribe()
-      } else if (user.loadingState === asyncStates.success) {
+      const { user } = getRoot(self)
+      if (user.loadingState === asyncStates.success) {
```

Comment:
```suggestion
      if (!user.isLoggedIn && user.loadingState === asyncStates.success) {
```

It’s been a while since I looked at the User code but I think this will fix it, and is easier to read.

Reply:
I like this more than my commit a minute ago, checking out more 👍

Reply:
Yep, `isLoggedIn` is a view from the User store returning `return !!self.id`, but I think makes more sense from a code read, so I'm going to update to use that (`isLoggedIn`) ✅ , just a sec...